Dockerfile

```diff
@@ -35,6 +35,8 @@ RUN apt-get update \
     unzip \
     locales \
     openssh-client \
+    # openssl is required to build Rubies < 3.1 on Ubuntu 22.04
+    openssl \
```

Comment:
This will fail because it simply installs `openssl` 3.x... when we need an older version of `openssl`, which simply isn't available via `apt`